google-cloud-bigquery/pom.xml

```diff
@@ -76,6 +76,10 @@
       <groupId>com.google.api</groupId>
       <artifactId>api-common</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.api-client</groupId>
+      <artifactId>google-api-client</artifactId>
```

Comment:
What is this new dependency used for?

Reply:
Previously it was a transitive dependency needed for the apiary client. In the test I added, one of the exceptions I throw is a Google Json exception, the same kind apiary clients produce. If I mark that dependency as `<scope>test</scope>` that removes it from the compile classpath and breaks the compile, so I had to add it as a non-test dependency.

Reply:
Can we simplify the test to now use this new dependency?

Reply:
changes to using Connect exception instead